### DISCUSSION TITLE
No Transcriptions

### DISCUSSION THREAD

**Discussion by iamnafets** [deepgram/community#125] [General help] [answered]
I'm currently using Nova with the nodejs client and experiencing high failure rates (>50%). Unfortunately, I get 0 error messages -- nothing is returned in the transcription. I've confirmed my code works (it does work sometimes).

![image](https://user-images.githubusercontent.com/93804/232184506-b49a92da-bc78-4dfc-b610-71ff332df6fa.png)

What else can I provide to help debug?

**Comment by iamnafets** [comment]
This may be related to the rate that I'm sending samples. I get much higher success (and ofc, much higher latency) by sending 1000ms time slices. It's possible there are message ordering problems?

**Reply by i-bee-ai** [reply]
@iamnafets - has the problem been resolved for you? I still have the same issue.

**Comment by SandraRodgers** [comment, marked as answer]
Hi @iamnafets 

We have a resource to help debug the request in this situation, however it is in Python. I highly recommend using this if you are not opposed to trying it out in Python. If you go through the steps in our [Streaming Test Suite ](https://developers.deepgram.com/documentation/guides/streaming-test-suite/), it will help you to check that everything is working. You clone the repo and walk through the steps with the sample audio to make sure everything works, and then you can try streaming your audio.

You could try this out if you have time. In the meantime, I will try to find out the answer you have about message ordering problems.
